sinks/signalfx/signalfx.go

```diff
@@ -280,3 +249,86 @@ func (sfx *SignalFxSink) SetExcludedTags(excludes []string) {
 	}
 	sfx.excludedTags = tagsSet
 }
+
+type ddSampleKind int
+
+const (
+	ddSampleEvent ddSampleKind = iota
+	ddSampleServiceCheck
+	ddSampleUnknown
+)
+
+func ddOtherSampleKind(tags map[string]string) ddSampleKind {
+	if _, ok := tags[dogstatsd.CheckIdentifierKey]; ok {
+		return ddSampleServiceCheck
+	}
+	if _, ok := tags[dogstatsd.EventIdentifierKey]; ok {
+		return ddSampleEvent
+	}
+	return ddSampleUnknown
+}
+
+func (sfx *SignalFxSink) convertServiceCheck(sample *ssf.SSFSample) error {
+	name := sample.Name
+	tags := map[string]string{}
+	for k, v := range sample.Tags {
+		if k != dogstatsd.CheckIdentifierKey {
+			tags[k] = v
+		}
+	}
+	tags[sfx.hostnameTag] = sfx.hostname
+	tags["veneursinkonly"] = sfx.Name()
+	for k, v := range sfx.commonDimensions {
+		tags[k] = v
+	}
```

Comment:
This will need an additional tag, `"veneursinkonly": sfx.Name()`, to prevent the metric from being reported to all sinks, i.e. datadog&sfx (we want it to go only into sfx).

Reply:
I think this tag gets added 4 lines above your comment. Am I misunderstanding your request?

Reply:
oh god, yep, I missed it. Carry on!

Reply:
Cool!